docs/userguide/main.md

```diff
@@ -1146,6 +1146,9 @@ The namespace of the `OpenSearchUser` must be the namespace the OpenSearch clust
 
 Note that a secret called `sample-user-password` will need to exist in the `default` namespace with the base64 encoded password in the `password` key.
 
+Also, it is possible to store multiple Users password in the same Secret. To do this, you should create a secret where
+each key will be equal to a username and value is a user password.
```

Comment:
Please rewrite this to make it clearer that if multiple passwords are stored in one secret it **must** be done this way because the operator expects it. Right now it sounds like a suggestion, but the way you have written the code it would not work correctly otherwise.

Reply:
done